projects/mysadna/firestore.rules

```diff
@@ -0,0 +1,7 @@
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow read, write;
+    }
+  }
+}
```

Comment:
You can delete `firestore.rules` and `firestore.indexes.json`.
Please don't forget to remove it inside `firebase.json` too